models/maat/R/write.config.MAAT.R

```diff
@@ -59,19 +59,19 @@ convert.samples.MAAT <- function(trait.samples, runid) {
   }
   if ("Ha.vcmax" %in% names(trait.samples)) {
     ## Convert from kJ mol-1 to J mol-1
-    trait.samples <- transform(trait.samples, Ha.vcmax = PEcAn.utils::ud_convert(Ha.vcmax, "kJ", "J"))
+    trait.samples$Ha.vcmax <- PEcAn.utils::ud_convert(trait.samples$Ha.vcmax, "kJ", "J")
```

Comment:
I'm amused that the `transform`'s in this file were obviously intended to "save typing" but most of these lines are shorter without it!